packages/react/src/primitives/SelectField/SelectField.tsx

```diff
@@ -10,6 +10,27 @@ import { SelectFieldProps, Primitive } from '../types';
 import { splitPrimitiveProps } from '../shared/styleUtils';
 import { useStableId } from '../shared/utils';
 
+const selectFieldChildren = (
+  children?: React.ReactNode,
+  options?: string[]
+) => {
+  if (options?.length) {
+    if (children) {
+      console.warn(
+        'Amplify UI: <SelectField> component defaults to rendering children over `options`. When using the `options` prop, do not also pass children.'
+      );
+    } else {
+      return options.map((option, index) => (
+        <option label={option} value={option} key={index}>
```

Comment:
Should we avoid using `index` as `key`?

Reply:
Yes, if possible. What else could we use as the `key`?

Reply:
How about combining the option value and index: `${option}-${index}`